TYPE - NEW SCRIPT TEMPLATE.vbs

```diff
@@ -67,7 +75,11 @@ DO
         Call validate_footer_month_entry(MAXIS_footer_month, MAXIS_footer_year, err_msg, "*")   'IF NEEDED
         'The rest of the mandatory handling here
         IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note." 'IF NEEDED
-        IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
+		If ButtonPressed = msg_show_instructions_btn Then
```

Comment:
Minor comment you use msg_show_instructions_btn  in the error handling and script_instructions_btn in your draft dialog btn var - might want to make them the same.

Reply:
Good catch!